Servers/UI/OJS.Servers.Ui/ClientApp/src/common/types.ts

```diff
@@ -68,19 +68,28 @@ interface ISubmissionResponseModel {
     processed: boolean;
 }
 
+interface IGetAllAdminParams {
```

Comment:
it is indeed, but we better address that in seperate PR, since it affects mainly administration service queries which doesnt have much to do with the context of these changes